mmdet/models/losses/dice_loss.py

```diff
@@ -0,0 +1,34 @@
+import torch.nn as nn
+
+from ..builder import LOSSES
+from .utils import weighted_loss
+
+
+@weighted_loss
+def dice_loss(pred, target):
+    x = pred
+    eps = 1e-5
```

Comment:
suggest making eps as an argument that can be set through the dice_loss interface and the DiceLoss class.